web_src/js/components/DashboardRepoList.vue

```diff
@@ -255,6 +256,32 @@ const sfc = {
     $(el).find('.dropdown').dropdown();
     nextTick(() => {
       this.$refs.search.focus();
+      this.$refs.search.addEventListener('keydown', (e) => {
+        const previousPageLink = this.$refs.previousPageLink;
+        const nextPageLink = this.$refs.nextPageLink;
+        switch (e.key) {
+          case 'Enter':
+            this.$refs[`repoIndex${this.repoIndex}`][0].click();
+            break;
+          case 'ArrowUp':
+            if (this.repoIndex > 0) this.repoIndex--;
+            else if (previousPageLink && !previousPageLink.classList.contains('disabled')) previousPageLink.click();
+            break;
+          case 'ArrowDown':
+            if (this.repoIndex < this.repos.length - 1) this.repoIndex++;
+            else if (nextPageLink && nextPageLink.classList && !nextPageLink.classList.contains('disabled')) {
```

Comment:
What is the purpose of checking for `.classList` existance?

Reply:
Also, use full if syntax with braces here and above, e.g.:

```js
if (cond) {
 ...
} else {
 ...
}

Reply:
> What is the purpose of checking for `.classList` existance?

Because too many repositories will have pagination, it is not allowed to click the next page when doing the next page